SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -6,8 +6,8 @@
 import FWCore.ParameterSet.Config as cms
 
 
-from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
-process = cms.Process('Phase2PixelNtuple',Phase2C9)
+from Configuration.Eras.Era_Phase2C8_cff import Phase2C8
```

Comment:
the change in this file and the file below constitute a regression
please remake this PR on the latest IB